examples/linux/pi-network.yml

```diff
@@ -0,0 +1,48 @@
+# Ensure nri-flex is installed in /var/db/newrelic-infra/newrelic-integrations/bin
+# Place pi-network.yaml in /etc/newrelic-infra/integrations.d and restart the newrelic-infra agent
+# Tested on an armv7l Rapsberry Pi
```

Comment:
[optional] might be nice to add a sample NRQL for both event types here
e.g.: `SELECT bus, device, id, description FROM usbStatusSample`

It's not very clear to see what the attribute names are from the `interfaceLinkSample` events

Reply:
Updated.